runtime/doc/starting.txt

```diff
@@ -104,9 +104,9 @@ rvim	 vim -Z	    Like "vim", but in restricted mode (see |-Z|)   *rvim*
 rview	 vim -RZ    Like "view", but in restricted mode.	  *rview*
 rgvim	 vim -gZ    Like "gvim", but in restricted mode.	  *rgvim*
 rgview	 vim -RgZ   Like "gview", but in restricted mode.	  *rgview*
-evim     vim -y     Easy Vim: set 'insertmode' (see |-y|)	  *evim*
-eview    vim -yR    Like "evim" in read-only mode		  *eview*
```

Comment:
I think your corrections to evim and eview are each missing a space after the first tab and before "vim".

Reply:
Thanks. I'll fix.